man/systemd.network.xml

```diff
@@ -2350,6 +2350,32 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
     </variablelist>
   </refsect1>
 
+  <refsect1>
+      <title>[DHCPServerStaticLease] Section Options</title>
+      <para>The <literal>[DHCPServerStaticLease]</literal> section configures a static DHCP lease
+      to assign a pre-set IPv4 address to a specific device based on its MAC address.
+      This section can be specified multiple times.
+      </para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>MACAddress=</varname></term>
+
+          <listitem><para>The hardware address of a device which should be assigned IPv4 address
+          specified in <varname>Address=</varname>. This key is mandatory.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><varname>Address=</varname></term>
+
+          <listitem><para>IPv4 address that should be assigned to a device with a hardware address
+          specified in <varname>MACAddress=</varname>. This key is mandatory.</para>
```

Comment:
This claims that the `Address` key is mandatory, but ` src/network/networkd-dhcp-server.c` line 390 claims:

```
                /* Static lease with an empty or omitted address is a valid entry,
                * it removes any static lease with the specified mac address. */
```